### PR TITLE
qemu.tests: Add boot_with_different_vectors test case and script.

### PR DESCRIPTION
**Pull request by FengYang**
Signed-off-by: Feng Yang <fyang@redhat.com>

### CODE REVIEW

File: qemu/tests/boot_with_different_vectors.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            if not (re.findall("virtio[0-9]-input", output) and
+                    re.findall("virtio[0-9]-output", output)):
+                msg = "Could not find the device for msi interrupt "
+                msg += "when vectors = %s " % vectors
+                msg += "Command %s got output %s" % (cmd, output)
+                error.TestFail(msg)
+
+    vectors_list = params["vectors_list"]
+    random.seed()
+    rand_num = random.randint(3, 7)
```
Comment (FengYang) [inline qemu/tests/boot_with_different_vectors.py]: Discuss with owner, auto script could test all the vectors from 0 to 11 and -1.
So update the script.

**Comment by xutian** [comment]
LGTM,

Acked-by: @xutian

**Comment by suqinhuang** [comment]
LGTM
Acked-by: Suqin Huang

**Comment by ypu** [comment]
Thanks @FengYang @xutian and @suqinhuang . Will merge it.

**Pull request merged**